### DISCUSSION TITLE
Doc comment on Google Cloud Marketplace (/installation/gc-marketplace)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1642] [Docs/blog comments]
# Doc comment on Google Cloud Marketplace (/installation/gc-marketplace)

Overview

https://weaviate.io/developers/weaviate/installation/gc-marketplace

**Comment by Battjmo** [comment]
I spun up one of these last Friday and was charged $143 in three days. Is that expected? I never actually loaded any data into it, just did a couple of test API calls and put it down for the weekend.

It appears to have been storing about 19TB of data from the very beginning. Is this expected? Possible I messed something up, I am not an expert in cloud deployments, but it seems very odd.

**Reply by robertlair-lifequest** [reply]
Has anyone else had this issue?  I really want to try this, but this comment makes me hesitant.

**Comment by jplotts** [comment]
I am similarly having what appear to be significant unexpected costs.  I spun up the cluster and did nothing with it, not even test requests.  The total costs have been $50+ in 30 hours.  Per the GCP billing page, Weaviate has calculated that I've used 8000 GB and 2000 CPU.  However, per GCP cluster resource page, I'm only using 8GB and 2CPU, a factor of 1000 less.  Can someone help explain the discrepancy?
